components/gems/Gemfile.lock

```diff
@@ -600,7 +605,7 @@ GEM
       mixlib-log
     mixlib-authentication (2.1.1)
     mixlib-cli (2.1.1)
-    mixlib-config (3.0.1)
```

Comment:
I cut https://github.com/markdownlint/markdownlint/pull/280 to prevent this from pulling down mixlib-config

Reply:
Merged, thanks!